sos/report/plugins/openvswitch.py

```diff
@@ -33,6 +33,31 @@ def setup(self):
             "ovs-vsctl -t 5 get Open_vSwitch . other_config:dpdk-init")
         check_dpdk = (dpdk_enabled["status"] == 0 and
                       dpdk_enabled["output"].startswith('"true"'))
+        check_6wind = any([self.is_installed(p) for p in
+                           ['6windgate-fp', 'nuage-openvswitch']])
+        actl = "ovs-appctl"
```

Comment:
Why introducing this variable, that is in fact a constant that makes the `add_cmd_output` commands less readable? Some preparation for conditional assignment of the variable in future?

I am not against it per se, just.. the code sounds less obvious with the constant variable.

ACK otherwise.

Reply:
That's because of the 80 column limit. It was easier to do this than try to split the strings IMHO.